README.md

```diff
@@ -1,9 +1,5 @@
 # curriculum-databases-projects-template
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
 ## Getting Started
```

Comment:
[Optional] I suggest that you tell user to use  **postgreSQL** for example for this project because when I use **SQLite**, the **create** query crashed (near "IDENTITY": syntax error)